### PR TITLE
circleci and github actions ci implementation

### PR DESCRIPTION
**Pull request by JJediny**
Changes approach to only use declared terraform and referencing terraformer for refrerencing imported resources only

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            fi
+      - run:
+          name: Validate Terraform config
+          command: |
+            if [[ $(echo "$CIRCLE_BRANCH" | grep -c "pull") -gt 0 ]]; then
+              echo "This is from a fork."
+              terraform validate terraform
+            else
+              terraform plan -input=false terraform
+            fi
```
Comment (afeld) [inline .circleci/config.yml:26]: If we end up having to repeat this config in a bunch of places, might make sense to [create an orb](https://circleci.com/docs/2.0/orb-intro/) for it (later).

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+version: "2.1"
+jobs:
+  plan:
+    docker:
+      - image: hashicorp/terraform:0.11.14
```
Comment (afeld) [inline .circleci/config.yml:5]: Is this version correct?

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    docker:
+      - image: python:3-alpine
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: pip install -r requirements.txt
+      - run:
+          name: Run validations
+          command: pytest tests
```
Comment (afeld) [inline .circleci/config.yml:54]: Assuming this was copied in? I think you can delete all of `validate`.

File: README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # TTS New Relic Automation
 
 ## Getting Started
-To protect against committing credentials or other secrets:
+To protect agaisnt committing credentials or other secrets:
```
Comment (afeld) [inline README.md:4]: You were right before 😉

File: README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
...
 - [Gitleaks](https://github.com/zricethezav/gitleaks#getting-started)
 - [Pre-commit](https://pre-commit.com/#install)
 - [Mozilla's Secrets OPerationS (SOPS)](https://github.com/mozilla/sops)
```
Comment (afeld) [inline README.md:7]: Is SOPS actually being used for this repository? If not, let's remove from the README to reduce confusion.

**Review by afeld — CHANGES_REQUESTED** [review]
General feedback:

- We were going to worry about continuous deployment separately/later—https://github.com/18F/tts-tech-portfolio/issues/631 is still in the Icebox. Since I don't think the CircleCI configuration will work as it stands, please delete it for now.
- What are all the `snapshot/` files? Do we still need them?

**Comment by its-a-lisa-at-work** [comment]
What is the current status of this pull request? @afeld do you still want the changes made by @JJediny ?

**Comment by afeld** [comment]
@JJediny If you think you can get the feedback addressed in under, say, an hour, I say just go for it and we can get this merged. Otherwise, let's close and it can be reopened later.

**Pull request closed without merging**